modules/ibi-generate-seed-image.adoc

```diff
@@ -0,0 +1,175 @@
+// Module included in the following assemblies:
+// * edge_computing/ztp-image-based-upgrade.adoc
+// * edge_computing/factory-image-based-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-seed-generation_{context}"]
+= Generating a seed image with the {lcao}
+
+Use the {lcao} to generate the seed image with the `SeedGenerator` CR. The Operator checks for required system configurations, performs any necessary system cleanup before generating the seed image, and launches the image generation. The seed image generation includes the following tasks:
+
+* Stopping cluster operators
+* Preparing the seed image configuration
+* Generating and pushing the seed image to the image repository specified in the `SeedGenerator` CR
+* Restoring cluster operators
+* Expiring seed cluster certificates
+* Generating new certificates for the seed cluster
+* Restoring and updates the `SeedGenerator` CR on the seed cluster
+
+[NOTE]
+====
+The generated seed image does not include any site-specific data.
+
+To avoid including any {rh-rhacm-first} resources in your seed image, you need to disable all optional {rh-rhacm} add-ons for seed image generation.
+====
+
+.Prerequisites
```

Comment:
Probably it's worth to mention that for precaching to work, the seed image needs to create an extra partition to be used for precaching

You can check in IBU documentation: https://access.redhat.com/articles/7057298#generating-a-seed-image-with-the-lifecycle-agent-12, specifically the prerequisites of the seed cluster used to create a seed image

Reply:
Thanks Javi! Does this mean a prerequisite for the seed cluster is something like "You must create an additional partition for precached container images"? 
And then this will be reflected in the seed image.

Reply:
Yes, that would be a prerequisite for precaching to work

Note that even if you don't want to use precaching for installation time using IBI, it's necessary if in the future you will want to upgrade the cluster using IBU with precaching